go.mod

```diff
@@ -3,364 +3,377 @@ module github.com/dapr/components-contrib
 go 1.19
 
 require (
-	cloud.google.com/go/datastore v1.8.0
```

Comment:
This change to dependencies seems to be reverting a lot of packages to an older version. Could you please reset all go.mod files to the master branch, then just do a “go mod tidy”?

Reply:
Changed as requested